matlab_simulation/code v1.0/ME597-5-ExtendedKalmanFilter/EKF.m

```diff
@@ -0,0 +1,24 @@
+function [mu,S,K,mup] = EKF(mu,S,w,v,dt,y,motion_model,sensor_model,linearize_motion_model,linearize_sensor_model,R,Q)
+%% Extended Kalman Filter.
+%This function takes mu, Covariance, motion and sensor model and their derivaties as input
+%and returns Kalman Gain, Updated mu & Covariance and predicted mu.
+
+
+%% Extended Kalman Filter Estimation
+    n=length(mu);
+    %y=sensor_model(mu);
+
+    % Linearization
+    Gt = linearize_motion_model(mu,v,dt);
+    Ht = linearize_sensor_model(mu);
+    
+    % Prediction
+    mup = motion_model(mu,w,v,dt);

```

Comment:
Where is this function located? I might be missing it, but looks like this function was deleted in this commit  a64bc77a0caa541b1cfd0b3947c77d315f5da8af,